temporalcli/commands.workflow_view.go

```diff
@@ -117,6 +117,8 @@ func (c *TemporalWorkflowDescribeCommand) run(cctx *CommandContext, args []strin
 		StateTransitionCount int64
 		HistoryLength        int64
 		HistorySize          int64
+		ParentExecution      *common.WorkflowExecution `cli:",cardOmitEmpty"`
+		RootExecution        *common.WorkflowExecution `cli:",cardOmitEmpty"`
```

Comment:
I think 4 fields of `ParentWorkflowId`, `ParentRunId`, `RootWorkflowId`, and `RootRunId` would look a bit better on text output here (all being strings and having `cardOmitEmpty` too)

Reply:
Sure, I can do that. Ideally we would make it easier to recursively text-ify a struct with indents, that would be ideal, but when I've tried that before it was more work than I wanted to invest at the moment.